robot/markers.py

```diff
@@ -168,9 +168,12 @@ def spherical(self) -> SphericalCoord:
         """
         return SphericalCoord(*self._raw_data['spherical'])
 
+    def __repr__(self):
+        return "<robot.markers.{} >".format(self.__str__())
```

Comment:
You shouldn't need to call `__str__` explicitly here.

Reply:
fixed in f6b302caa0584e038006c46df4e43ae474f7910e